launch_ros/launch_ros/actions/load_composable_nodes.py

```diff
@@ -55,15 +58,10 @@ def __init__(
         :param composable_node_descriptions: descriptions of composable nodes to be loaded
         :param target_container: the container to load the nodes into
         """
-        ensure_argument_type(
-            target_container,
-            ComposableNodeContainer,
-            'target_container',
-            'LoadComposableNodes'
-        )
```

Comment:
@eric1221bday why dropping the type check? `ensure_argument_type` can take more than one type.

Reply:
@hidmic It's because it has trouble with Union types, which is what SubstitutionType ultimately is. Alternative we can list out every single type a substitution can possibly be, but that felt very hacky to me.

Reply:
You can use `SomeSubstitutionsType_types_tuple`, e.g.:

https://github.com/ros2/launch/blob/8f1ed681973aad9b48ecc6712e54d2f66ee4dac3/launch/launch/actions/timer_action.py#L66

Reply:
@wjwwood added it back in with suggested tuple, seems to work